src/llama.cpp

```diff
@@ -9127,7 +9127,7 @@ static int llama_model_load(const std::string & fname, llama_model & model, llam
             )
         )) {
             // TODO(cebtenzzre): propagate this error outside of llama_load_model_from_file
-            LLAMA_LOG_WARN("%s: disabling Kompute due to unsupported model arch or quantization\n", __func__);
+            LLAMA_LOG_WARN("%s: disabling Kompute due to unsupported model arch (%s) or quantization (%s)\n", __func__, LLM_ARCH_NAMES.at(model.arch), llama_model_ftype_name(model.ftype).c_str());
```

Comment:
This warning will be removed in #10026, since the llama model loader will be able to offload weights only to backends that can actually use them, which should make this unnecessary. However, that PR will also remove support for backends that do not implement the backend reg and device interfaces, and currently only Kompute doesn't. If there is interest in keeping this backend in llama.cpp, it should be updated to implement these interfaces.

Reply:
Oh ok I can close this